lua/minuet/blink.lua

```diff
@@ -85,8 +85,24 @@ function M:get_completions(ctx, callback)
 
             local items = {}
             for _, result in ipairs(new_data) do
+                local line_entry = vim.split(result, '\n')
+                local item_label = nil
+                if #line_entry == 1 then
+                    item_label = result .. ' [single]'
+                else
+                    for _, line in ipairs(line_entry) do
+                        line = utils.remove_spaces({ line })[1]
```

Comment:
we don't need remove spaces for each line in completion items.

Suppose when there is a completion items containing if-else condition, remove the spaces will result in worse readability